package.json

```diff
@@ -9,9 +9,9 @@
     "test": "test"
   },
   "scripts": {
-    "prepush": "grunt quicklint && cd mailer && npm run prepush",
-    "postinstall": "cd mailer && npm install",
-    "shrinkwrap": "npmshrink",
+    "prepush": "grunt quicklint && grunt templates && git status -s | (! grep 'M lib/senders/templates/')",
+    "postinstall": "scripts/download_l10n.sh",
+    "shrinkwrap": "npmshrink && npm run postinstall",
```

Comment:
Without this explicit `postinstall` the mailer fails after you regenerate shrinkwrap.